src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -634,9 +663,10 @@ private bool TryGetVariableStyle(
             }
 
             private bool IsWrittenInsideForFrameworkValueType(
-                Dictionary<ISymbol, List<SyntaxToken>> symbolMap, ISymbol symbol, bool writtenInside)
+                MultiDictionary<ISymbol, SyntaxToken> symbolMap, ISymbol symbol, bool writtenInside)
             {
-                if (!symbolMap.TryGetValue(symbol, out var tokens))
+                var tokens = symbolMap[symbol];
```

Comment:
>var tokens = symbolMap[symbol]; [](http://example.com/codeflow?start=16&length=31)

Guess I haven't used MultiDictionary enough, threw me for a quick loop that the indexer doesn't throw if the key isn't present.

Reply:
fair.  but yeah, that is the semantics.  it maps 'not there' to 'empty'